metadata/update_metadata_features.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import json
+import os
+import pandas as pd
+
+
+def main(feature: str, domain: str, feature_dir: str, metadata_file: str):
+    metadata = json.load(metadata_file)
+    for filename in os.listdir(feature_dir):
+        f = os.path.join(feature_dir, filename)
+        if f.endswith(".parquet"):
+            # for simple features that we want to aggregate, pandas should work fine
+            feature_df = pd.read_parquet(f)
+            feature_mean = feature_df[feature].mean()
+            feature_std = feature_df[feature].std()
+            metadata["features"][f"{feature}_mean"] = feature_mean
```

Comment:
I think we need to change the dataset metadata schema to add this as a feature, but out of interest of time, will just add that as a TODO when we actually run this